code/game/objects/structures/crates_lockers/closets/job_closets.dm

```diff
@@ -73,23 +73,38 @@
 /obj/structure/closet/jcloset/New()
 	..()
 	new /obj/item/clothing/under/rank/janitor(src)
```

Comment:
Theres now 32 items here. A closet's capacity is 30. Please reduce the item count.

Reply:
I knew I should have tested closing the locker...